### PR TITLE
feat: Add express example

### PR DESCRIPTION
**Pull request by igneel64**
### Description
Add a simple TypeScript - Express.js example

### CODE REVIEW

File: express/src/tracking/snowplow.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {
+  tracker,
+  gotEmitter,
+  HttpProtocol,
+  HttpMethod,
+} from "@snowplow/node-tracker";
+
+const emitter = gotEmitter("0.0.0.0", HttpProtocol.HTTP, 9090, HttpMethod.GET);
```
Comment (matus-tomlein) [inline express/src/tracking/snowplow.ts:8]: We should rather use `POST` here as users might be inspired by the example and copy the setting.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,7 +0,0 @@
-{
```
Comment (matus-tomlein) [inline .vscode/settings.json:1]: This was my crap that I added to Git by mistake 😅 Thanks for cleaning up!

**Review by matus-tomlein — APPROVED** [review]
Awesome, so good to finally have a NodeJS example!

File: express/src/tracking/snowplow.ts (hunk 1 above)
Comment (igneel64) [reply express/src/tracking/snowplow.ts:8]: The micro collector does not work with HTTPS, that is why I added that 😆 
Do you think we should change it ?

Comment (matus-tomlein) [reply express/src/tracking/snowplow.ts:8]: You can use Micro over HTTPS when using an [ngrok tunnel](https://ngrok.com/). I don't think it's that important here as people will use whatever they have. I think the protocol is a bit more important to show correctly as users might not have a preference/don't know the advantages of GET vs POST in our tracker implementation.

Comment (igneel64) [reply express/src/tracking/snowplow.ts:8]: Thank you, great explanation! Adding it now!

**Pull request merged**